processes/VOTING.md

```diff
@@ -1,26 +1,40 @@
 # Technical Steering Committee Voting Process
 
 While the Project aims to operate as a consensus-based community, many decisions
-will require a vote in order to move forward:approval of new repositories
+will require a vote in order to move forward: approval of new repositories
 or subprojects, approval of archival/abandonment of repositories or subprojects,
 approval of new maintainers, selection of Project Lead, etc.
 
-Voting will happen either within a meeting, or via online vote.
+Voting will happen either within a meeting, or via electronic vote.
 
-Meetings will happen on the **{to be determined day of the month}**. An agenda
-will be provided as an issue on the issue tracker of this repository, with the
-ability for anybody to request an item be added, modified, or removed.
+For purposes of this document, when 50% results in a fraction, `ceil(50%)` is
+implied.
```

Comment:
when 50% is not in fraction are we using 50% + 1, or just 50%?

Reply:
Just 50%, no + 1. This was agreed in the meeting.